website/client/app.vue

```diff
@@ -399,6 +408,7 @@ export default {
       if (this.selectedSpellToBuy.pinType === 'card') {
         const newUserGp = castResult.data.data.user.stats.gp;
         this.$store.state.user.data.stats.gp = newUserGp;
+        this.text(this.$t('sentCardToUser', { userName: member.profile.name }));
```

Comment:
Could this be changed from `userName` to `profileName` here and in the translatable string? That would avoid any uncertainty when someone is viewing the locales file about whether it's the profile name or login name. Sorry about this - I realise I wrote "User Name" in the issue. I should have been using better terminology.